README.md

```diff
@@ -55,16 +63,12 @@ Now restart your app and you should see data in your SolarWinds dashboard in a m
 
 ## Important!
 
-`solarwinds-apm` should be the first file required. If, for example, you are using the `esm`
-package to enable ES module syntax (import rather than require) and you use the following
-command to invoke your program `node -r esm index.js` then `esm.js` is loaded first and
-`solarwinds-apm` is unable to instrument modules. You can use it, just make sure to require
-`solarwinds-apm` first, e.g., `node -r solarwinds-apm -r esm index.js`.
+`solarwinds-apm` should be the first file required. 
+
+If, for example, you are using the `esm` package to enable ES module syntax (import rather than require) and you use the following
+command to invoke your program `node -r esm index.js` then `esm.js` is loaded first and `solarwinds-apm` is unable to instrument modules. You can use it, just make sure to require `solarwinds-apm` first, e.g., `node -r solarwinds-apm -r esm index.js`.
 
-If you are using the custom instrumentation SDK then a reference to the SDK must be obtained, 
-like `const apm = require('solarwinds-apm')`. It you  use the command line option to require
-`solarwinds-apm` (e.g. `node -r solarwinds-apm -r esm index.js`) you can access the SDK using
-`const apm = global[Symbol.for('solarwinds-apm')]`
+If you are using the custom instrumentation SDK then a reference to the SDK must be obtained,  like `const apm = require('solarwinds-apm')`. If you use the command line option to require `solarwinds-apm` (e.g. `node -r solarwinds-apm -r esm index.js`) you can access the SDK using `const apm = global[Symbol.for('solarwinds-apm')]`
```

Comment:
> However, as more and more users use a transpile & build process to create their Node apps, the require (or import) method no longer works since the bundler (a la webpack) is unable to work with the bindings binaries.

Is this correct or not?: Users that transpile & build for Node apps (_not_ vanilla JavaScript) cannot get the agent to work if they only `require('solarwinds-apm')` and they don't launch the app with `node -r solarwinds-apm <app.js>`. This is what I've gathered from PR description but I'm not sure 🙂 

If the above is correct, I think that L57 should be `Custom Instrumentation SDK` instead of `Build Time`, and
L59 should be 

```
global[Symbol.for('solarwinds-apm')]
// an example call to SDK
```

If I'm wrong, please disregard!

Reply:
You got it right. 
But since there may be situations where user needs to use the `require` at the top without obtaining reference and using the SDK (e.g. user can't change the start command), it should be there in simple form. 

How about we call them "At Start" and "In Code"?

Reply:
Yes sure! That's more clear to me.

Reply:
@tammy-baylis-swi - clarified more. Take a look... thanks!!